### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by u0251077** [bryandlee/animegan2-pytorch#18]
What is the loss function used by the "Face Portrait v2" model? The effect is amazing.

**Comment by bryandlee** [comment]
L2 distance + LPIPS + Adversarial loss were used for the generator

**Comment by baixinping618** [comment]
> “Face Portrait v2”模型使用的损失函数是什么？效果是惊人的。

Can you reproduce the training of cartoon faces?

**Comment by baixinping618** [comment]
> 生成器使用 L2 距离 + LPIPS + 对抗性损失

How many weights are these loss functions?
